fakeserver/routes.go

```diff
@@ -128,6 +128,11 @@ func (s *server) routes() {
 		"/api/v1/split_details/{id}",
 		getV1SplitDetail,
 	)
+
+	s.handleGet(
+		"/api/v3/builds/{timeStamp}/split_registry",
```

Comment:
nit: above it looks like we were calling the build timestamp property in other URLs `{b}` - not that opaque one letter abbreviations are better in almost any circumstances, but I think I justified it in my head because some of the lines were very long and the parameters were largely being thrown away anyway, so giving them throwaway parameter names seemed revealing.